app/Modules/Group/Models/Group.php

```diff
@@ -244,7 +254,17 @@ public function getIsScvcepAttribute(): bool
 
     public function getIsVcepOrScvcepAttribute(): bool
     {
-        return $this->isVcep || $this->isScvcep;
+        return $this->getCuratesVariantPathogenicity;
```

Comment:
probably should be `$this->getCuratesVariants();`, or else rename the other below.